### PR TITLE
Footer react

### LINKED ISSUE #1040: JSX: footer

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-07-18 at 09 25 51

Closes #1040

### CODE REVIEW

File: src/components/text/Link.jsx [hunk 1]
```diff
@@ -18,7 +18,8 @@ const SIZE = {
   OBSCURE: 'obscure'
 };
 
-const Link = ({children, href = '#', color, size = SIZE.NORMAL, unstyled, underlined, emphasised, disabled}) => {
+const Link = ({children, href = '#', color, size = SIZE.NORMAL, unstyled, underlined, emphasised, disabled,
```
Comment (aju) [inline src/components/text/Link.jsx]: Maybe we should refactor it in the way that if link does not have href set then its disabled?

Comment (aju) [inline src/components/text/Link.jsx]: @vergilius @kdzwinel

Comment (anna-wro) [inline src/components/text/Link.jsx]: @kdzwinel, @matzimowski, @burakukula, @matikrk, @vergilius what do you think about it?

**Comment by matzimowski** [comment]
@anna-wro imho, we decided to stay with `sg-footer` as `<footer>`, not `sg-layout__footer`. we gonna remove `sg-layout__footer` in further steps. could you just replace `<footer>` in `<div>` again in layout component (not to have `<footer><footer></footer></footer>`?

**Comment by anna-wro** [comment]
@matzimowski  so `<Footer>` would be 

```
<footer className="sg-footer">
  <div className="sg-footer__container">
    {children}
  </div>
</footer>;
```

and in `<Layout>`
`footerContent = <div className="sg-layout__footer">{footer}</div>;` ? Is that correct?

**Comment by matzimowski** [comment]
for now yes. It would be correct, then I will refactor layout and get rid of useless wrapper

**Pull request merged**